build/teamcity-publish-artifacts.sh

```diff
@@ -9,7 +9,7 @@ export BUILDER_HIDE_GOPATH_SRC=1
 
 build/teamcity-publish-s3-binaries.sh "$@"
 
-if [ "$TC_BUILD_BRANCH" != master ]; then
+if [ "$TEAMCITY_BUILDCONF_NAME" == 'Publish Releases' ]; then
```

Comment:
Won't this still trigger for alpha releases, judging by https://teamcity.cockroachdb.com/viewType.html?buildTypeId=Cockroach_ReleaseBuild?